src/libsyntax/ext/expand.rs

```diff
@@ -154,7 +154,34 @@ ast_fragments! {
         "impl item"; many fn flat_map_impl_item; fn visit_impl_item; fn make_impl_items;
```

Comment:
@c410-f3r 
Did the `impl<'a, 'b> MutVisitor for MacroExpander<'a, 'b>` impl above required any changes in visitors?

(It is not really necessary, only a couple of methods from it is used, and it may be simpler to just remove it if it causes inconveniences.)

Reply:
Addressed in https://github.com/rust-lang/rust/pull/63537.